src/core/transaction.js

```diff
@@ -1112,6 +1112,77 @@ Transaction.prototype.setDeliverMin = function(deliver_min) {
   return this._setAmount('DeliverMin', deliver_min);
 };
 
+Transaction.prototype.suspendedPayment = function(options_) {
+  const options = _.merge({}, options_);
+
+  if (_.isUndefined(options.cancel_after)) {
```

Comment:
It's an option rather than a method name, but I could be convinced to use whateverCaseThisIs for everything